### PR TITLE
more RA config suggestions

### PR DESCRIPTION
**Pull request by RalfJung**
Cc https://github.com/rust-analyzer/rust-analyzer/issues/8667 @matklad

### CODE REVIEW

File: src/building/suggested.md [hunk 1]
```diff
@@ -34,7 +34,9 @@ you can write:
     "rust-analyzer.rustfmt.overrideCommand": [
       "./build/TARGET_TRIPLE/stage0/bin/rustfmt"
     ],
-    "editor.formatOnSave": true
+    "editor.formatOnSave": true,
+    "rust-analyzer.cargo.runBuildScripts": false,
+    "rust-analyzer.procMacro.enable": false,
```
Comment (jyn514) [inline src/building/suggested.md:39]: Doesn't JSON disallow trailing commas?
```suggestion
    "rust-analyzer.procMacro.enable": false
```

**Review by jyn514 — APPROVED** [review]
Thanks!

File: src/building/suggested.md (hunk 1 above)
Comment (RalfJung) [reply src/building/suggested.md:39]: It works for me in vscode configs...

**Comment by RalfJung** [comment]
Weird, this branch was merged but my commit does not show up in the repo? This is confusing the heck out of my "detect which branches can be deleted" tooling.^^

**Comment by jyn514** [comment]
Hmm, I think I might have used "squash and merge" instead of rebasing it. Not sure if there's  a way  to detect that automatically.

**Comment by RalfJung** [comment]
Ah yeah that would explain it. One more reason I prefer proper merges. :) But obviously I cannot tell you how to run a repo you maintain.

**Pull request merged**